e2e/protractor/suites/search/search-filters.test.ts

```diff
@@ -335,13 +335,13 @@ describe('Search filters', () => {
       expect(await dataTable.isItemPresent(fileJpgUser1.name)).toBe(true, 'JPG file not displayed');
       expect(await locationFilter.getChipTitle()).toEqual(site, 'Incorrect location filter selected');
 
-      await locationFilter.openDialog();
-      await locationFilter.setAutocompleteInputValue('_REPOSITORY_');
-      await locationFilter.clickApplyButton();
-
-      expect(await dataTable.isItemPresent(filePdfUser2.name)).toBe(true, 'PDF file not displayed');
-      expect(await dataTable.isItemPresent(fileJpgUser1.name)).toBe(true, 'JPG file not displayed');
-      expect(await locationFilter.getChipTitle()).toEqual(`${site}, _REPOSITORY_`, 'Incorrect location filter selected');
+      // TODO check this again after ES migration works as expected when tested manually
```

Comment:
should we raise some jira/add comment to existing jira to don't forget about this?

Reply:
Yes I will raise it today, thanks!